TESTS/host_tests/ticker_timing.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
This host test seems to have the similar functionality as timing_drift_auto.py. It might be best to use that test, as timing_drift_auto.py is able to determine if the timing failure is due to CI latency. CC @studavekar

Reply:
It looks like timing_drift_auto.py can be used to check if time passed on the board is valid (consistent with time passing on host). Here we need to verify if ticker frequency declared by vendor is valid. Host test provides pattern 10s delay time and on the device number of ticks during this period is counted. Then check is performed if counted number of ticks is consistent with declared frequency with 10% accuracy.
I could wait 10s on target using wait(), but this function is based on ticker, so using a tested feature in the test might not be a good solution.

Reply:
You can accurately validate the frequency by using timing_drift_auto.py if you convert ticks to microseconds and sent it back to the host test. I would highly recommend using timing_drift_auto.py as this was specifically designed to handle latency problems that we have seen in the past.